README.md

```diff
@@ -4,6 +4,12 @@ This repository contains the source code for Haiku's main website.
 
 This website is generated using [Hugo](https://gohugo.io).
 
+## Issue Tracker
+We track all issues on our own instance of [Trac](https://dev.haiku-os.org).
+
+To view all the tickets that involve this repository, just
+[click here](https://dev.haiku-os.org/query?component=%5EWebsite%2Fwww.haiku-os.org&status=assigned&status=closed&status=in-progress&status=new&status=reopened&order=priority&group=milestone).
```

Comment:
do we really need the "status=closed" ones listed? If not, remove that. If yes, you don't need to list all the status and can remove them all from the URL.

Reply:
I think that was accidental -- I used the UI to filter them and then changed my mind after I had done so.

Reply:
Fixed.